pkg/api/vlabs/types.go

```diff
@@ -128,9 +128,10 @@ const (
 
 // OrchestratorProfile contains Orchestrator properties
 type OrchestratorProfile struct {
-	OrchestratorType    string            `json:"orchestratorType" validate:"required"`
-	OrchestratorVersion string            `json:"orchestratorVersion"`
-	KubernetesConfig    *KubernetesConfig `json:"kubernetesConfig,omitempty"`
+	OrchestratorType        string            `json:"orchestratorType" validate:"required"`
+	OrchestratorVersionHint string            `json:"orchestratorVersionHint"`
```

Comment:
This will be a breaking change in the public (preview) API version. Are we ok with this?